easybuild/easyconfigs/i/iccifort/iccifort-2019.4.227-GCC-8.3.0.eb

```diff
@@ -0,0 +1,42 @@
+# This is an easyconfig file for EasyBuild, see http://easybuilders.github.io/easybuild
+
+name = 'iccifort'
+version = '2019.4.227'
+versionsuffix = '-GCC-8.3.0'
```

Comment:
If we are dropping the binutils suffix because it's always just one version anyway; can't we go all the way here and drop the entire suffix as the same applies for iccifort -> GCC.

Reply:
Makes sense, see also discussion in https://github.com/easybuilders/easybuild-easyconfigs/pull/8681/files#r311570349